es-vue-base/src/lib-mixins/forms.js

```diff
@@ -5,8 +5,9 @@ export default {
     data() {
         return {
             submitInProgress: false,
-            formMsg: '',
-            formMsgVariant: 'danger',
+            formShowSuccess: false,
+            formShowError: false,
+            formMsgVariant: '',
```

Comment:
We should keep `'danger'` as the initial value for `formMsgVariant`, otherwise an empty string could be passed in, which would fail the prop validation set up [here](https://github.com/EnergySage/es-ds/pull/967/files#diff-c8e866b38f24e2b2f5834114c4ff3f537ab64df09137064efa1add7e42ec0025R49) and generate a warning in the console. The default value would still be applied but worth avoiding potentially confusing console warnings for people trying to use the default forms mixin behavior.